tempto-core/src/main/java/com/teradata/tempto/internal/query/QueryResultValueComparator.java

```diff
@@ -196,12 +203,21 @@ private static int floatingEqualWithTolerance(Object actual, Object expected, do
 
     private int singleFloatingEqual(Object actual, Object expected)
     {
-        return floatingEqualWithTolerance(actual, expected, MACHINE_EPSILON_FLOATING_32B);
+        return floatingEqualWithTolerance(actual, expected, getFloatTolerance(MACHINE_EPSILON_FLOATING_32B));
```

Comment:
remove MACHINE_EPSILON_FLOATING_32B - as those seem to not make sense as they do not depend on actual/expected value.